rmw_connextdds_common/src/common/rmw_impl.cpp

```diff
@@ -180,7 +180,7 @@ rmw_connextdds_parse_string_list(
       DDS_String_free(*el_ref);
     }
     *el_ref = DDS_String_alloc(next_len);
-    if (nullptr == el_ref) {
+    if (nullptr == *el_ref) {
```

Comment:
This could be right...? I want to double check because it seems unrelated to the changes for this feature.

Reply:
It's indeed unrelated to the rest of the PR, but I was looking through this code for reference and noticed that it wasn't checking the correct pointer here, so I thought I'd slip in this fix.